charts/portworx/templates/portworx-rbac-config.yaml

```diff
@@ -23,13 +23,10 @@ rules:
     verbs: ["delete", "get", "list", "watch", "update"]
   - apiGroups: [""]
     resources: ["persistentvolumeclaims", "persistentvolumes"]
-    verbs: ["get", "list", "create", "delete", "update"]
```

Comment:
These helm-charts _do_ add several more RBAC-permissions compared to the install.portworx.com YAML-charts.
This was done to avoid using `{if} / {else} / {end}` -blocks, and/or introducing new configuration variables -- so we'd define _all permissions_ instead of being selective via `{if} / {else} / {end}` -blocks.

Is this going to be a problem  (defining extra RBAC permissions?)

Reply:
I see here permissions are removed, not defined, or there is something different you're referring to?

Reply:
Oh.. yes, that's correct -- we have removed permissions here, and this should now be "in sync" with the install.portworx.com 

Example: https://install.portworx.com/?mc=false&kbver=1.19.3&k=etcd%3Ahttp%3A%2F%2F70.0.95.189%3A2379&m=eth1&d=eth1&c=px-cluster-123&stork=true&st=k8s&csi=true

```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
   name: node-get-put-list-role
rules: ...
- apiGroups: [""]
  resources: ["persistentvolumeclaims", "persistentvolumes"]
  verbs: ["get", "list"]
```

Reply:
ah, got it. If we wait with this approval long enough, no changes will be needed since this spec matches 2.8 :)
https://edge-install.portworx.com/2.8?mc=false&kbver=&b=true&c=oksana-cluster-e5401f2b-0a4a-4035-8067-4aca5d3073cd&stork=true&csi=true&st=k8s

Reply:
Oh -- so you're saying I should re-synchronize RBAC-permissions again, but this time using px-2.8.0 as a reference?
Not sure if this would be appropriate, since my ticket was asking to re-sync the RBAC's against latest px-2.7.x

Reply:
Let's only match with `/2.7` for now since that was the intention of the ticket.

Reply:
@harsh-px in this case we can't use helm chart for tanzu installations

Reply:
Let's do that as part of a separate change. No need to bundle everything in one PR. PWX-20255 was to sync specs with PX 2.7.